superset-frontend/src/explore/controls.jsx

```diff
@@ -191,7 +191,7 @@ const jsFunctionInfo = (
   </div>
 );
 
-function columnChoices(datasource) {
+export function columnChoices(datasource) {
```

Comment:
@kristw just pointing out this export/import... we may want to move this to `utils` or somewhere else that makes more sense.

Reply:
Agree. We can separate this function into its own file starting in this PR too.

Reply:
I'll make another PR moving this (and probably some other things) into separate files/modules. I'll track it in my internal documents for now.